layouts/announcements.hbs

```diff
@@ -11,25 +11,72 @@
       {{> navigation key='foundation'}}
 
       <article>
-        <div class="container no-headline">
-
+        <div class="container no-headline" id="announcements-list">
           <ul class="news-list">
-            {{#each collections.blogAnnounce}}
-              {{#if title}}
-                <li>
-                  <time datetime="{{ date }}">{{ strftime date }}</time>
-                  <a href="/{{../site.locale}}/{{ path }}/">{{ title }}</a>
-                </li>
-              {{/if}}
-            {{/each}}
+            <li>Loading data...</li>
           </ul>
-
         </div>
       </article>
-
     </div>
   </div>
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/tabletop.js/1.4.2/tabletop.min.js"></script>
+  <script type="text/javascript">
+  var public_spreadsheet_url = 'https://docs.google.com/spreadsheets/d/1xwQ6ciZLADL43My_XBIWLVRuDMl8cx_omYd1FRkBZ58/pubhtml';
```

Comment:
We use the standard.js style across the nodejs.org website, please make sure client side scripts follow the same style.

Reply:
It was copied from [layouts/in-the-news.hbs#L22](https://github.com/nodejs/nodejs.org/master/layouts/in-the-news.hbs#L22) to use layout as news sections. If something goes wrong I'll fix it on both pages.

Reply:
It was copied from [layouts/in-the-news.hbs#L22](https://github.com/nodejs/nodejs.org/blob/master/layouts/in-the-news.hbs#L22) to use layout as news sections. If something goes wrong I'll fix it on both pages.

Reply:
@pierreneter The code below now has semicolons at the end, for example, which is not the case for `in-the-news.hbs` but had other issues fixed in #1352.

Could you please fix that so it validates against standard style?
See https://standardjs.com/demo.html for a quick check.

Reply:
Yes. I will push a commit tonight